Automattic/jetpack
# Replace WP Logo links that point to about/contribute pages on Simple Sites and Atomic

Fixes https://github.com/Automattic/dotcom-forge/issues/7671

## Proposed changes:

* Replace the  wp-admin/about.php and wp-admin/contribute.php links on Simple Sites since we restrict access to these pages
* The links would be removed from Atomic too for consistency reasons

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply these changes on your Simple Site and on an Atomic site
* Go to WP-Admin and click on the "W" logo on the top-right of the page (admin-bar)
* You should be redirected to the main WP-Admin page instead of the about.php page.
